magento/magento2
# Add Imagick CMYK to SRGB conversion

Add Imagick CMYK to SRGB conversion

### Description (*)
When Imagick adapter detects that the opened image has a CMYK colorspace, convert the colorspace to SRGB.

### Related Pull Requests

1. magento/magento2#25277

### Fixed Issues (if relevant)

1. Fixes magento/magento2#22375

### Manual testing scenarios (*)

1. Set Imagick as the image adapter
2. Create a product and configure the following file: 
![image](https://user-images.githubusercontent.com/20220341/56230294-a1eb1b00-6084-11e9-9f00-2cd8fcadc2ed.jpg)
3. The colors on the frontend of the resized image should be correct.

### Questions or comments
Not sure if the CMYK image added to the unit test has any copyrights to it. @ioweb-gr Can you confirm if the image can be used for this purpose?

### Contribution checklist (*)
